### DISCUSSION TITLE
HookConfig, EndpointConfig, etc no longer exported in directus npm package

### DISCUSSION THREAD

**Discussion by dwene** [directus/directus#12030] [Archive]
### Describe the Bug

Looks like the the typescript interfaces used for building Hooks and Endpoints are no longer being exported in 9.6.0

`import { HookConfig } from 'directus/dist/types';` import does not exist.

### To Reproduce

Try creating a typescript file and importing the below

`import { HookConfig } from 'directus/dist/types';`

### What version of Directus are you using?

9.6.0

**Comment by sensedrive** [comment]
@dwene

I am not familiar with the API/Code changes during the last releases. Does it help you to use the following import:

```
import { HookConfig } from '@directus/shared/types'
```

**Comment by licitdev** [comment]
> ```
> import { HookConfig } from '@directus/shared/types'
> ```

Thank you @sensedrive, that is right. Ref: https://github.com/directus/directus/pull/11191
